UniWatch/Views/Lecture/Override.cshtml

```diff
@@ -22,6 +22,50 @@
         <input type="submit" value="Update" class="btn btn-success" />
         @Html.ActionLink("Delete", "Delete", "Lecture", new { lectureId = Model.Lecture.Id }, new { @class = "btn btn-danger" })
         <p />
+        if (Model.Lecture.Images.Any())
+        {
+            <div id="divCarousel" class="carousel slide" data-ride="carousel" data-interval="false">
+                <!-- Indicators -->
+                <ol class="carousel-indicators">
+                    <li data-target="#divCarousel" data-slide-to="0" class="active"></li>
+                    @for (var i = 1; i < Model.Lecture.Images.Count; i++)
+                    {
+                        <li data-target="#divCarousel" data-slide-to="@i"></li>
+                    }
+                </ol>
+
+                <!-- Wrapper for slides -->
+                <div class="carousel-inner" role="listbox">
+                    <div class="item active">
+                        @{ var image = Model.Lecture.Images.ElementAt(0); }
+                        <img src="@image.Url" alt="@string.Format($"Lecture Image #{image.Id}")">
+                    </div>
+
+                    @for (var i = 1; i < Model.Lecture.Images.Count; i++)
```

Comment:
The first item in the carousel is marked active. Rather than checking every element in the for loop if it is the first one, the first element is marked active and the rest (if there are any) are added to the carousel.